api/src/test/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessorTest.java

```diff
@@ -57,44 +58,22 @@
     @InjectMocks
     private DetermineEntitlementMessageProcessor processor;
 
-    @Test
-    void shouldSuccessfullyProcessMessageAndTriggerPaymentForEligibleClaimant() {
+    @ParameterizedTest
+    @EnumSource(EligibilityStatus.class)
+    void shouldSuccessfullyProcessMessageAndTriggerPaymentForEligibleClaimant(EligibilityStatus eligibilityStatus) {
```

Comment:
I'm not sure I like the two original tests being condensed into one to save a few lines of duplicated code in this case, I'd much rather see one test with all the statuses apart from INELIGIBLE and one test for that scenario. Also, the test method name is now no longer correct.

Reply:
I've renamed the test to be more clear but wouldn't want to move away from the enum source. This way we correctly test all the of negative cases instead of just INELIGIBLE.